app/lib/src/ui/components/terms/content_screen.dart

```diff
@@ -1,7 +1,9 @@
+import 'package:auto_route/auto_route.dart';
 import 'package:flutter/material.dart';
 import 'package:mondaymorning/src/services/themes/mm_colors.dart';
 import 'package:mondaymorning/src/ui/components/terms/terms_content.dart';
 
+@RoutePage<dynamic>(name: 'ContentRoute')
```

Comment:
Why dynamic? This page doesn't return any data so it should be void instead.


```suggestion
@RoutePage<void>(name: 'ContentRoute')
```